src/api/courseOperations.tsx

```diff
@@ -0,0 +1,41 @@
+import { Course } from 'api';
+import { AddCourseData } from 'components/AddCourseModal';
+
+const handleErrors = (response: Response) => {
+	if (!response.ok) {
+		throw Error('fetch unsuccessful');
+	}
+	return response;
+};
+
+// create
+export const fetchPostCourse = async (data: AddCourseData) =>
```

Comment:
Let's use names like `createCourse`, `getCourse`, `updateCourse` and `deleteCourse` or something you think is better but still reads very clearly what it does